h/static/styles/variables.scss

```diff
@@ -42,6 +42,21 @@ $success-color: #1cbd41 !default;
 $mask-start-color: rgba($white, 0) !default;
 $mask-end-color: $white !default;
 
+
+// Variables for the new color palette
+// ------------------------------------
+$grey-1: #F2F2F2;
```

Comment:
I'll do that as part of a clean up right after I restyle the annotation card and the replies.